integration-tests/pkg/types/runtime_config.go

```diff
@@ -0,0 +1,13 @@
+package types
+
+type RuntimeConfig struct {
+	Networking struct {
+		ExternalIps struct {
+			Enable bool
+		}
+	}
```

Comment:
just a note that we'll need to add `PerContainerRateLimit` (probably in a follow up PR)

Reply:
We should not be defining a RuntimeConfig struct here. We should be importing the go code generated from the protobuf definitions. That should be done in a different PR.